index.js

```diff
@@ -600,9 +600,9 @@ class ClientServiceBase {
   ) {
     const request = await this._createContractExecutionRequest(
```

Comment:
the private function _createContractExecutionRequest has changed the argument order in 3.5 and this part was missed to be revised because of the lack of unit-test.

Reply:
@supl Thank you for the fix. This is not directly related to this fix and I'm not sure but `createSerializedContractExecutionRequest()` seems a deprecated-style interface. Can users call it with a new style when using serialized binaries?

Reply:
@jnmt 
Ah, Thank you, Jun.
It's a good catch.
Sorry, I thought I had implemented another function already but I hadn't done it.
Let me create another PR for that.
Thanks for your reminder!

Reply:
@jnmt 
I implemented it in #88 but because #88 is based on the branch of this PR, so I will set you as reviewer after this PR is merged.